tests/unit/test_alertmanager_client.py

```diff
@@ -59,3 +60,23 @@ def test_version(self, urlopen_mock):
         urlopen_mock.return_value.reason = "OK"
 
         self.assertEqual(self.api.version, "0.1.2")
+
+    @patch("alertmanager_client.urllib.request.urlopen")
+    def test_alerts_can_be_set(self, urlopen_mock):
```

Comment:
Looks like a whitebox test. Do we need it?

Reply:
As I understood the task was to test that templates can be set and that these templates lead to customized notifications being sent to the receiver. That is what is being tested here.